site/src/atomics/width.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Width Atomics
+description: Width related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Width Atomics
+
+Need to specifiy a specific width for an element? You've come to the right place! These classes should be used sparingly, and only used after testing heavily at all screen widths (starting with mobile). That said, if you need to specify and min, max or normal width, they can be a useful tool in the utility belt.
+
+Values are in pixels. Keep in mind that the `min-width` and `max-width` properties override the standard `width` property. If you aren't using the Atlas core folder as a base for you styles, these classes work best when [you're setting `box-sizing` to be `border-box` in some way](https://css-tricks.com/box-sizing/#aa-present-day-box-sizing).
+
+| cssproperty                                      | value (px)                               | screensize                                         |
+| ------------------------------------------------ | ---------------------------------------- | -------------------------------------------------- |
+| `width`                                          | `100`, `150`, `200`, `250`, `300`, `350` | all screensizes, `tablet`, `desktop`, `widescreen` |
+| `width` (available on tablet screens and larger) | `400`, `450`                             | `tablet`, `desktop`, `widescreen`                  |
+
+## Usage
+
+Set the `width` of an element.
+
+```html
+<div class="width-100 border padding-sm font-weight-semibold">
+	<p>100</p>
+</div>
+<div class="width-150 border padding-sm font-weight-semibold">
+	<p>150</p>
+</div>
+<div class="width-200 border padding-sm font-weight-semibold">
+	<p>200</p>
+</div>
+<div class="width-250 border padding-sm font-weight-semibold">
+	<p>250</p>
+</div>
+<div class="width-300 border padding-sm font-weight-semibold">
+	<p>300</p>
+</div>
+<div class="width-350 border padding-sm font-weight-semibold">
+	<p>350</p>
+</div>
+```
+
+### Responsive rules
+
+Certain widths are too large for many mobile screens, so they are only available on larger screensizes. However, all helpers can be applied to any screensize.
```

Comment:
_However, all helpers can be applied to any screen size._ - I understand what you meant here, however, I think it might sound a little bit confusing at a first glance, cause not _all_ helpers can be applied to _any_ screen size. For example, I might think about `width-400` as to be available on all screen sizes including mobile but it doesn't exist without `{screensize}` prefix actually.

Reply:
this is gone now

Reply:
I still see the same thing.

Reply:
🤦‍♂️Apologies @ollips, yes, I see it here now. Your point is definitely correct, this sentence is misleading.